### PR TITLE
[lab-intro-to-ml]Samuel Nnoruka

### PR DESCRIPTION
**Pull request by SamiNn007**

### CODE REVIEW

**Comment by ta-data-lis** [comment]
Lab checked, well done! ✅

**In the cell below, convert the Date column values from string to numeric values using toordinal()**
```
from datetime import datetime

def convert_date_to_ordinal(date):
    return datetime.strptime(date, '%Y-%M-%d').toordinal()

austin_fixed['Date'] = austin_fixed['Date'].apply(convert_date_to_ordinal)
```

**separate the data into a training set and a test set using the train_test_split()**
```
y = austin_fixed['TempAvgF']
X = austin_fixed.drop(columns=['TempAvgF'])
X_train, X_test, y_train, y_test = train_test_split(X,y, test_size= 0.20)
```
